cmd/virt-handler/Dockerfile

```diff
@@ -23,9 +23,20 @@ FROM centos:7
 # no-lower than what was used when the binary was built.
 RUN yum -y install wget && \
     wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
-    && yum -y install libvirt-client \
+    && yum -y install libvirt-client cyrus-sasl-md5 \
```

Comment:
I tried to install on libvirtd and also on virt-handler containers cyrus-sasl-scram package only, but when I tried to connect to the libvirt I received the error:
`Failed to start SASL negotiation: -4 (SASL (-4) : no mechanism available: No worthy mechs found)`